app/FreshRSS.php

```diff
@@ -141,6 +141,10 @@ public static function loadStylesAndScripts(): void {
 						FreshRSS_View::prependStyle(Minz_Url::display(FreshRSS::getThemeFileUrl($theme_id, $filename)));
 				}
 			}
+
+			if (!empty($theme['theme-color'])) {
```

Comment:
opinion: using empty() is an bad pratice 

```
"" (an empty string)
0 (0 as an integer)
0.0 (0 as a float)
"0" (0 as a string)
NULL
FALSE
array() (an empty array)
```
Return false.
they not a real empty. i suggest use `array_keys_exist()` instead of.

Reply:
I personally prefer `empty()` here, which is exactly for what we need: check that the value exists and that it is not falsy.
Not tested recently, but `array_key_exists()` is also slower.

Reply:
Although the empty() function is useful for checking if a variable is empty, it can have some drawbacks. Here are some arguments against using empty():

Unexpected behavior with certain data types: The empty() function can produce unexpected results with certain data types, including numerical values, strings that contain digits, strings containing spaces or zeros, etc. For example, empty('0') returns true, which can be confusing.
Ambiguity with boolean values: When used with boolean values, the empty() function can produce unexpected results. For example, empty(false) returns true, which can be confusing.
Poor code readability: Using empty() can make code less readable, as it requires a double negation to get the opposite result. For example, if (!empty($var)) { ... } may be more difficult to understand than if (isset($var)) { ... }.
Limitations for checking array keys: The empty() function cannot be used to check if a specific key exists in an array. For this task, you should use array_key_exists().
Risk of modifying the original variable: The empty() function modifies the original variable by converting it to a boolean type, which can cause unintended side effects in your code.
In summary, although the empty() function is useful in some situations, it is important to understand its limitations and consider other options when writing more complex code.

Reply:
None of that applies there, as far as I can see